### DISCUSSION TITLE
Sidekiq 7 connection refused on listing RetrySet and ScheduledSet

### DISCUSSION THREAD

**Discussion by satriahu** [sidekiq/sidekiq#5986] [Q&A] [answered]
hi. I am using `Sidekiq 7.1.2` and `Sidekiq Pro 7.1.3` and Redislabs (`Redis 7.2.0`) as redis server.
I found an issue when visiting `Retries` and `Scheduled` tab in Sidekiq Admin Dashboard (Rails 6). as attached below.
![image](https://github.com/sidekiq/sidekiq/assets/81661682/760a4e94-03d7-47c5-ad06-95e6950a0d95)

when I tried rails console to see what the problem is. and i found this : 
![image](https://github.com/sidekiq/sidekiq/assets/81661682/c6b3d493-a9fa-44e2-a786-29076f1b34f1)

my connection was good on the first command `Sidekiq::Queue.new.first`.
but when I tried accessing Retry Set or Scheduled Set, the connection was refused and all connections (including other Rails Server instances) were also cut and refused for around 3 seconds until it recovered.

I also tried sidekiq with redis on my local machine, and it was fine.
![image](https://github.com/sidekiq/sidekiq/assets/81661682/f1f6b028-ba57-48e3-8ec8-fcaf8c16fd19)

Can anybody please help me?

**Comment by mperham** [comment, marked as answer]
Redis labs does not yet support Sidekiq 7, this is their own limitation. You can use 6 or another Redis provider until they’ve fixed their issue.

**Reply by satriahu** [reply]
Thank you for clarifying !
